src/features/setup-device/SetupDevice.tsx

```diff
@@ -9,14 +9,16 @@ function NewSession() {
   const [parentName, setParentName] = React.useState<string>("");
   const [childName, setChildName] = React.useState<string>("");
   const [treatment, setTreatment] = React.useState<Treatment | "">("");
+  const [testType, setTestType] = React.useState<string>("");
   const dispatch = useDispatch();
 
   const disabled = !parentName || !childName || !treatment;
 
   const onSubmit = (e: React.FormEvent) => {
     e.preventDefault();
+    // if (childName && parentName && treatment && testType) { //empty string in Javascript is technically "falsy" therefore && testType == false
     if (childName && parentName && treatment) {
-      dispatch(startSession({ childName, parentName, treatment }));
+      dispatch(startSession({ childName, parentName, treatment, testType }));
```

Comment:
Do you know if Test Type should be a require field? 

Since you didn't change line `20`, currently looks like they would be able to submit the form without specifying a test type. 

I'm not quite sure how Mary wants to roll this out. If she has test type assignments ready for everyone going forward, then maybe we should make it required? Although we would want to make sure she knows that the implication is that as soon as this is merged people could no longer get through set up without a test type.